Rock.Messaging.RabbitMQ/IRabbitSessionConfiguration.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Rock.Messaging.RabbitMQ
+{
+    public interface IRabbitSessionConfiguration
```

Comment:
re: Setters. Yes, they ought to be removed from other implementations (note to self: do that). An object that consumes a configuration object has no reason to change configuration values, so a getter-only config interface is the way to go.